websites/Y/Yandex/presence.ts

```diff
@@ -0,0 +1,126 @@
+const presence = new Presence({
+		clientId: "1035433850593095740",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	homepageInput =
+		document.querySelector<HTMLInputElement>('input[name="text"]'),
+	{ host, pathname } = document.location;
+
+let strings: Awaited<ReturnType<typeof getStrings>>;
+
+enum Assets {
+	logo = "https://i.imgur.com/SFXKtLF.png",
+	search = "https://i.imgur.com/wYVlwJX.png",
+	view = "https://i.imgur.com/hxvvGUi.png",
+}
+
+async function getStrings() {
+	return presence.getStrings({
+		search: "general.search",
+		view: "general.viewing",
+	});
+}
```

Comment:
If you are using strings, you should also have a multiLang setting in the metadata.json